addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3647,6 +3647,9 @@ export class OdooEditor extends EventTarget {
             const selection = this.document.getSelection();
             // Find previous character.
             let { focusNode, focusOffset } = selection;
+            if (!focusNode) {
+                return;
+            }
```

Comment:
Could you add steps to reproduce the issue to the commit message ? Because it's not obvious to me how you can manage to have the focus inside the editor without having a selection. I just tried pressing tab around the page to focus the editable without clicking but it still gave me a selection.